### PR TITLE
Resource Alert Policy 

### PR DESCRIPTION
**Pull request by msg4sunny**

### CODE REVIEW

File: opsgenie/resource_opsgenie_alert_policy.go [hunk 1]
```diff
@@ -0,0 +1,760 @@
...
+				Optional: true,
+				Default:  false,
+			},
+			"details": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				//Set: schema.HashString,
```
Comment (ffahri) [inline opsgenie/resource_opsgenie_alert_policy.go]: please delete unnecessary comments

File: opsgenie/resource_opsgenie_alert_policy_test.go [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+		_, err = client.GetAlertPolicy(context.Background(), &req)
+		if err != nil {
+			return fmt.Errorf("Bad: Alert policy %q does not exist", id)
+		}
+		return nil
+	}
+}
+
+func testAccOpsGenieAlertPolicy_basic(alertPolicyName string) string {
```
Comment (ffahri) [inline opsgenie/resource_opsgenie_alert_policy_test.go]: could you add another test function since this has a lot of fields we dont want to miss anything

**Review by ffahri — CHANGES_REQUESTED** [review]
tests are green

File: opsgenie/resource_opsgenie_alert_policy.go (hunk 1 above)
Comment (msg4sunny) [reply opsgenie/resource_opsgenie_alert_policy.go]: Done

File: opsgenie/resource_opsgenie_alert_policy_test.go (hunk 2 above)
Comment (msg4sunny) [reply opsgenie/resource_opsgenie_alert_policy_test.go]: Added test cases with other feields

**Pull request closed without merging**